src/odemis/driver/test/smaract_test.py

```diff
@@ -649,6 +652,56 @@ def test_auto_update_function(self):
         pos_after_move = self.dev.position.value
         testing.assert_pos_almost_equal(expected_pos, pos_after_move, **COMP_ARGS)
 
+    def test_param_file_nofile(self):
+        # check if forcing a ValueError is the result of a file that does not exist
+        with self.assertRaises(ValueError):
+            CONFIG_3DOF['param_file'] = "nonexistingfile.mcs2.tsv"
+            smaract.MCS2(**CONFIG_3DOF)
+
+    def test_parse_tsv_config(self):
+        # create a temporary .mcs2.tsv file
+        file_id, file_dir = tempfile.mkstemp(suffix='mcs2.tsv', text=True)
```

Comment:
`file_dir` sounds like the "file directory", but it's actually the completely file path (including the file name). So let's call it differently. Maybe `file_path` or `file_name`?